pynicotine/search.py

```diff
@@ -132,9 +132,7 @@ def remove_all_searches(self):
     def show_search(self, token):
         events.emit("show-search", token)
 
-    def process_search_term(self, search_term, mode, room=None, user=None):
-
-        users = []
+    def process_search_term(self, search_term, mode, room=None, users=None):
```

Comment:
What is the reason for `user` -> `users`?

Reply:
Was easier to deal with the internal support for multiple users this way, but on second thought, I don't know why we still support this when it's never been used. It should probably be removed.

Reply:
Saving this for a separate PR in any case.